src/widget.ts

```diff
@@ -46,6 +46,19 @@ export enum ElementWidgetActions {
   // host -> Element Call telling EC to stop screen sharing, or that
   // the user cancelled when selecting a source after a ScreenshareRequest
   ScreenshareStop = "io.element.screenshare_stop",
+  // This can be sent as form or to widget
```

Comment:
As EC automagically sets the muted state sometimes (based on room size? 🤔) it's important to be able to read the current state (without affecting it) when the CallKit session is established so that we keep the buttons in sync. How can we do that?

Reply:
```
  // (it is possible to sent an empty configuration
  // -> this will allow the client to only get the current state)
```
So you would send a `io.element.device_mute` action with the content `{}` then the repli would be:
```
response: {
 audio_enabled: true, // for example
 video_enabled: false // for example
}
```
with the format:
```
  // The data of the widget action request and the response are:
  // {
  //   audio_enabled?: boolean,
  //   video_enabled?: boolean
  // }
```

How could this be phrased to make more sense?

Reply:
Oh I see, perhaps `It is possible to sent an empty configuration to retrieve the current values or just one of the fields to update that particular value`

And I just noticed `An undefined field means that EC will not keep the mute state as is.`, I don't think that `not` belongs there, does it?